src/main/java/springboot/Model/StudentGroup.java

```diff
@@ -1,38 +1,37 @@
-package Model;
+package springboot.Model;
 
 import javax.persistence.*;
-import java.util.Collection;
-import java.util.HashSet;
+import java.util.ArrayList;
 import java.util.List;
 
 @Entity
-public class Group {
```

Comment:
we can just call it a group, the problem was with mysql so it's not an issue anymore

Reply:
Ok